ppocr/data/det/east_process.py

```diff
@@ -52,7 +52,7 @@ def convert_label_infor(self, label_infor):
         label_infor = label_infor.decode()
         label_infor = label_infor.encode('utf-8').decode('utf-8-sig')
         substr = label_infor.strip("\n").split("\t")
-        img_path = self.img_set_dir + substr[0]
+        img_path = self.img_set_dir + "/" + substr[0]
```

Comment:
The function `os.path.join()` is a preference.

Reply:
Done
there is case :  path1=os.path.join("aaa/bbb/","ccc") ,the result is :"aaa/bbb/","ccc"
 path2 = os.path.join("aaa/bbb/","/ccc")   ,the result is :"ccc"
path3 = os.path.join("aaa/bbb","/ccc")   ,the result is :"ccc"
so the image path in the label could not be start with a file separator